cli/manage.go

```diff
@@ -120,19 +119,11 @@ func manage(c *cli.Context) {
 
 	sched := scheduler.New(s, fs)
 
-	hb, err := strconv.ParseUint(c.String("heartbeat"), 0, 32)
-	if hb < 1 || err != nil {
-		log.Fatal("--heartbeat should be an unsigned integer and greater than 0")
-	}
-	options := &cluster.Options{
-		TLSConfig:       tlsConfig,
-		OvercommitRatio: c.Float64("overcommit"),
-		Discovery:       dflag,
-		Heartbeat:       hb,
+	cluster, err := swarm.NewCluster(sched, store, tlsConfig, append(c.StringSlice("cluster-opt"), "swarm.discovery="+dflag))
```

Comment:
yes, we could add another argument to NewCluster to pass it if you think it's better.